src/classes/activity-builder-activity/built-activity-handler.ts

```diff
@@ -583,7 +580,7 @@ export class BuiltActivityHandler implements ChatLogSubscriber {
     // handle sending prompt
     const requestFunction = async () => {
       const _response = await this.executePrompt(aiPromptSteps);
-      const response = extractServiceStepResponse(_response, 0);
+      const response = _response.answer;
```

Comment:
this is the `answer` property, so probably there's no need to call the var `response` & `_response`. Just a comment but an optional change.